src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, a) {
+      this.result = callback(this.result, a);
+
+      return this;
+    },
+
+    add(b, a) {
+      return b + a;
+    },
+
+    subtract(b, a) {
+      return b - a;
+    },
+
+    multiply(b, a) {
+      return b * a;
+    },
+
+    divide(b, a) {
+      return b / a;
+    },
```

Comment:
You need to add checking by 0, because you cannot divide by 0

Reply:
divide(b, a) {
      if (a !== 0) {
        return b / a;
      }
    },

Correct checking?